pkg/operator/ceph/nfs/config.go

```diff
@@ -48,12 +49,17 @@ func getNFSUserID(nodeID string) string {
 	return fmt.Sprintf("nfs-ganesha.%s", nodeID)
 }
 
-func getNFSClientID(n *cephv1.CephNFS, name string) string {
-	return fmt.Sprintf("client.%s", getNFSUserID(getNFSNodeID(n, name)))
+func getNFSClientID(name string) string {
```

Comment:
It will be useful to update PR description about the NFS version that has introduced this change.